ArenaControl/Stage3.cpp

```diff
@@ -264,7 +267,7 @@ static void addDigit(void)
 #endif
 
    /* If this is the first digit, the digit is just the number of turns */
-   if (prevPosition < 0) {
+   if (prevPosition == -100) {
```

Comment:
Also good catch - used 4269 as a tribute to Hitchhiker's Guide to the Galaxy (did you know 6 * 9 = 42 in base 13?)